Header header
uint8 size
uint32 frame_width
uint32 frame_height
Detection[] detections
